.appveyor.yml

```diff
@@ -0,0 +1,42 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: C:\projects\silverstripe
+
+services:
+  - mssql2014
+
+before_test:
+  - sqlcmd -S "(local)\SQL2014" -Q "Use [master]; CREATE DATABASE [appveyor]"
+  - SET PATH=C:\Program Files\OpenSSL;C:\tools\php;%PATH%
+  - cinst -y php -version 5.6.11
+  - cd c:\tools\php
+  - ps: cat php.ini-production | %{$_ -replace "memory_limit = 128M","memory_limit = 256M"} | Out-File -Encoding "Default" php.ini
+  - echo date.timezone="UTC" >> php.ini
+  - echo extension_dir=ext >> php.ini
+  - echo extension=php_openssl.dll >> php.ini
+  - echo extension=php_mbstring.dll >> php.ini
+  - echo extension=php_curl.dll >> php.ini
+  - echo extension=php_gd2.dll >> php.ini
+  - echo extension=php_tidy.dll >> php.ini
+  - php -r "readfile('http://getcomposer.org/installer');" | php
+  - php -r "readfile('https://dl.dropboxusercontent.com/u/7129062/sqlsrv_unofficial_3.0.2.2.zip');" > sqlsrv.zip
```

Comment:
* I can't find a working build of PHP 5.4 for Windows on Chocolatey.
 * The SQLSRV drivers were hard to find—I had to track down a custom build of them that supported 5.6.

It's possible that shifting to the standard windows versions of 3.2 might be worthwhile, but I don't think it blocks merge.